vllm/engine/llm_engine.py

```diff
@@ -1047,6 +1050,7 @@ def add_request(
         lora_request: Optional[LoRARequest] = None,
         trace_headers: Optional[Mapping[str, str]] = None,
         prompt_adapter_request: Optional[PromptAdapterRequest] = None,
+        priority: Optional[int] = None,
```

Comment:
Add to docstring, mention only applicable when priority scheduling mode is enabled.